### PR TITLE
run_mojo: Add --mojo-release and --mojo-debug flags.

### PR DESCRIPTION
**Pull request by nlacasse**
The flags control which build of mojo to run (Debug or Release).
Default is Release.

### CODE REVIEW

**Comment by nlacasse** [comment]
cc @jamesr

File: lib/src/run_mojo.dart [hunk 1]
```diff
@@ -42,12 +45,14 @@ class RunMojoCommand extends Command {
...
     List<String> args = [
       '--android',
-      '--release',
       'http://app/$appName',
       '--map-origin=http://app/=$appDir',
       '--map-origin=http://sky_viewer/=$skyViewerUrl',
       '--url-mappings=mojo:sky_viewer=http://sky_viewer/sky_viewer.mojo',
     ];
+    if (!argResults['mojo-debug']) {
+      args.add('--release');
```
Comment (jamesr) [inline lib/src/run_mojo.dart]: please explicitly add either --debug or --release to the mojo_run command line.

File: lib/src/run_mojo.dart [hunk 2]
```diff
@@ -42,12 +45,14 @@ class RunMojoCommand extends Command {
     String appDir = path.dirname(appPath);
     List<String> args = [
       '--android',
-      '--release',
```
Comment (jamesr) [inline lib/src/run_mojo.dart]: i think the command line is a bit cleaner if this is here and not at the end. i'd compute a string that is either "--debug" or "--release" based on the argResults and insert it here

**Comment by jamesr** [comment]
LGTM w/ comments

File: lib/src/run_mojo.dart (hunk 1 above)
Comment (nlacasse) [inline lib/src/run_mojo.dart]: Done.

File: lib/src/run_mojo.dart (hunk 2 above)
Comment (nlacasse) [inline lib/src/run_mojo.dart]: Done.

**Pull request merged**